psi4/src/psi4/libmints/basisset.cc

```diff
@@ -571,133 +574,122 @@ std::shared_ptr <BasisSet> BasisSet::zero_ao_basis_set()
     return new_basis;
 }
 
-std::shared_ptr<BasisSet>
-BasisSet::construct_ecp_from_pydict(std::shared_ptr <Molecule> mol, py::dict pybs, const int forced_puream){
+std::shared_ptr<BasisSet> BasisSet::construct_from_pydict(const std::shared_ptr <Molecule> &mol, py::dict pybs, const int forced_puream){
 
     std::string key = pybs["key"].cast<std::string>();
     std::string name = pybs["name"].cast<std::string>();
     std::string label = pybs["blend"].cast<std::string>();
-    std::string message = pybs["message"].cast<std::string>();
```

Comment:
[This](https://github.com/psi4/psi4/blob/master/psi4/driver/qcdb/libmintsbasisset.py#L824) line results in [this](https://github.com/psi4/psi4/blob/master/tests/scf-ecp/output.ref#L67) output, to tell the user where the ECP was found.  I'm currently assuming that ECPs and bases live together.